docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc

```diff
@@ -92,7 +92,7 @@ ifndef::add-copy-button-to-env-var[]
 Environment variable: `+++QUARKUS_KMS_DEVSERVICES_SERVICE_NAME+++`
 endif::add-copy-button-to-env-var[]
 --|string 
-|`localstack`
+|required icon:exclamation-circle[title=Configuration property is required]
```

Comment:
Hum. There is something extremely weird here with the config doc being changed that way.

Reply:
Though the same thing and think it was due to a previous bump version

Reply:
Issue with my vscode I guess. clean up my m2 registry and build outside vscode did revert changes.